site/content/docs/5.3/utilities/borders.md

```diff
@@ -122,6 +129,10 @@ Or, choose from any of the `.border-opacity` utilities:
 
 ## Width
 
+{{< design-callout-alert >}}
+The border variant with **a width of 3px** should not be used because it does not respect the Orange Design System specifications.
```

Comment:
```suggestion
The **`.border-3` variant** should not be used because it does not respect the Orange Design System specifications.
```

Reply:
OK for your proposal. My message is formulated like several other design callout messages where some of the illustrated variants shall be indicated.

Reply:
Haven't seen it before but there are 5 variants in Bootstrap (see https://twbs-bootstrap.netlify.app/docs/5.3/utilities/borders/#width), so we should reintegrate the 5 and modify the callout to say that 3, 4 and 5 shouldn't be used.
Do you mind if I add a commit to reintegrate these border utilities?

Reply:
Done via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2127

Reply:
```suggestion
The **`.border-{3|4|5}` variants** should not be used because it does not respect the Orange Design System specifications.
```